### ISSUE TITLE
Change selection at <key>+click, oder doubleClick

### ISSUE THREAD

**Issue by Moosbueffel** [thomasnordquist/MQTT-Explorer#496]
I use WIN10 and set to select an entry with only one click (like in Linux?). Perhaps this ist the problem I get with your wonderful MQTT Explorer, because the selection of a new topic will be done if I only move my cursor/mouse. So have to have hard attention what is selected. And sometimes I send my publish to a wrong destination because of this.

It would be better, if I have to click for selection, or press e.g. Shift+click or what else to change the selected item. And that it is not done automatically, if the mouse is over an entry.

Thanks

**Comment by mccaffm** [comment]
I agree with this, i thought it was just me, i selct on the left then have to be really careful not to stray into other fields beofre i move to the right, to delte entries for example. Often end up deleiting the wrong thing
